models/object.py

```diff
@@ -202,6 +209,7 @@ def setCursorShape(self, e_pos: QPoint):
             self.mode = Mode.MOVE
 
     def mouseReleaseEvent(self, e: QMouseEvent):
+        self.parentWidget().autosaver.onChangeMade()
```

Comment:
this would be a better place for registering something to undo, I shouldnt have but the autosave thing in that move event function, it gets ran for every pixel the object moves, it was kind of lagging if you moved stuff fast